app/views/audits/show.html.erb

```diff
@@ -84,10 +84,10 @@
             <td><%= name.humanize %></td>
             <% if @audit.action == 'update' %>
               <% change.each do |v| %>
-                <td><%= id_to_label(name,v,false) %></td>
+                <td><%= id_to_label(name, v, :truncate => false, :audit => @audit) %></td>
               <% end %>
             <% else %>
-              <td><%= id_to_label(name,change,false) %></td>
+              <td><%= id_to_label(name, change, :truncate => false, :audit => @audit) %></td>
```

Comment:
no need to pass the `:audit` parameter here and above since it's the default

Reply:
Same as https://github.com/theforeman/foreman/pull/5857#discussion_r207165668 , if I change one value of the options hash, I have to send the whole thing.

Quick demo:

```ruby
2.4.0 :001 > def test(opts = { :audit => 'hello', :truncate => true })
2.4.0 :002?>   puts opts
2.4.0 :003?>   end
 => :test
2.4.0 :004 > test
{:audit=>"hello", :truncate=>true}
 => nil
2.4.0 :005 > test :audit => 'bye'
{:audit=>'bye'}
```

Reply:
Oh, I guess i missed that, I thought you were using kwargs for the parameters, so you can just override one. Perhaps that would be simpler? 
```
[1] pry(main)> def foo(a: 'bla', b: 'gah')
[1] pry(main)*   puts a,b
[1] pry(main)* end  
=> :foo
[2] pry(main)> foo
bla
gah
=> nil
[3] pry(main)> foo(a: 'bar')
bar
gah
=> nil
[4] pry(main)> foo(b: 'baz')
bla
baz
=> nil